p/themes/Fresh-Nord/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Nord theme",
```

Comment:
Could you please use the same name than the directory? It has an impact on the alphabetic order in the gallery of themes.
So either *Fresh-Nord...* here, or change the name  (`git mv`) of the folder to *Nord...*

Reply:
Done, thanks for the reminder!